TensorIO/Classes/TensorFlow/TIOTensorFlowData/TIOPixelBuffer+TIOTensorFlowData.mm

```diff
@@ -22,6 +22,9 @@
 #import "TIOPixelBufferLayerDescription.h"
 #import "TIOVisionPipeline.h"
 
+//  TODO: switch to .flat mapping and copy bytes the same way they are for TFLite
+//  TODO: support batched and unbatched pixel buffers
```

Comment:
https://github.com/doc-ai/tensorio-ios/issues/41 and https://github.com/doc-ai/tensorio-ios/issues/38